src/pages/studyView/StudyViewPageStore.ts

```diff
@@ -1095,6 +1087,15 @@ export class StudyViewPageStore {
                                 sampleIdentifiers.length > 0
                             );
                         })
+                        .sortBy(attrVal => -attrVal.count)
```

Comment:
so essence of bug was that we were slicing before sort and therefor throwing out desired elements?

Reply:
exactly